### ISSUE TITLE
Flip able door. make so we can flip the door on the printers, so it opens the other way :-) 

### ISSUE THREAD

**Issue by bwinddk** [bambulab/BambuStudio#3758]
All of this is not needed just read the title it´s self explanatory :-D

Other than that thx for a great printer. i will mod my X1C so the door can open the other way, at some point when i get the time. :-)

Kind regards from Denmark :-) 

ps. i don´t need a response....

**Comment by GuzziRaz** [comment]
Yeah this isn't quite the right place for suggestions like this but I agree. And I think Bambu misses many opportunities on after sales. I would gladly order a kit moving the hinges to the right hand side, and I'd pay an overprice for it if it's a Bambu original part. Another thing many people would probably pay a lot for (I would) is a "Quiet premium fans kit" with drop-in replacements for the fans or an "Improved LED ramp" that is guaranteed not to fry your AP board. And so on.
